pyaedt/modules/Material.py

```diff
@@ -1353,46 +1356,60 @@ def __init__(self, materiallib, name, props=None, material_update=True):
 
     @property
     def material_appearance(self):
-        """Material Appearance specified as an RGB list.
+        """Material appearance specified as a list where first 3 items are rgb color and fourth one is transparency.
 
         Returns
         -------
         list
-            Color of the material in RGB.  Values are in the range ``[0, 255]``.
+            Color of the material in RGB and transparency.
+            Color Values are in the range ``[0, 255]``.
+            Transparency is float in the range ``[0,1]``
 
         Examples
         --------
-        Create a new material with color ``[0, 153, 153]`` (darker cyan).
+        Create a new material with color ``[0, 153, 153]`` (darker cyan) and transparency 0.5.
 
         >>> from pyaedt import Hfss
         >>> hfss = Hfss(specified_version="2021.2")
         >>> mat1 = hfss.materials.add_material("new_material")
-        >>> rgbcolor = mat1.material_appearance
-        >>> mat1.material_appearance = [0, 153, 153]
+        >>> appearance_props = mat1.material_appearance
+        >>> mat1.material_appearance = [0, 153, 153, 0.5]
         """
         return self._material_appearance
 
     @material_appearance.setter
-    def material_appearance(self, rgb):
-        if not isinstance(rgb, (list, tuple)):
-            raise TypeError("`material_apperance` must be a list or tuple")
-        if len(rgb) != 3:
-            raise ValueError("`material_appearance` must be three items (RGB)")
-        value_int = []
-        for rgb_item in rgb:
-            rgb_int = int(rgb_item)
-            if rgb_int < 0 or rgb_int > 255:
-                raise ValueError("RGB value must be between 0 and 255")
-            value_int.append(rgb_int)
-        self._material_appearance = value_int
+    def material_appearance(self, appearance_props):
+        if not isinstance(appearance_props, (list, tuple)):
+            raise TypeError("`material_appearance` must be a list or tuple")
```

Comment:
```suggestion
            raise TypeError("`material_appearance` must be a list or tuple.")
```
Message should be a complete sentence with concluding period.